dali/pipeline/data/tensor_test.cc

```diff
@@ -196,6 +197,7 @@ TYPED_TEST(TensorTest, TestGetBytesTypeSizeAlloc) {
   ASSERT_EQ(t.shape(), shape);
   ASSERT_TRUE(IsType<double>(t.type()));
   ASSERT_FALSE(t.shares_data());
+  delete [] ptr;
```

Comment:
would rather make an `std::vector<float>` in line 169 and forget about deleting it

Reply:
Done